texthero/preprocessing.py

```diff
@@ -614,3 +616,54 @@ def remove_urls(s: pd.Series) -> pd.Series:
     """
 
     return replace_urls(s, " ")
+
+
+def correct_mistakes(s: pd.Series, fix_spacing=False) -> pd.Series:
+    r""" Correct spelling mistakes
+
+    `correct_mistakes` finds all spelling mistakes from the given Pandas Series and replace them with suggested words.
+
+    When `fix_spacing` is set to ´True´, word_segmentation is applied.
+
+    Parameters
+    ----------
+    s : Pandas Series
+    fix_spacing : bool
+        When set to True, word_segmentation is applied, spacing is fixed in noisy text. Only use it if your data is similar to 'thequickbrownfoxjumpsoverthelazydog' as it slows performance
+    
+    Returns
+    -------
+    Pandas Series
+
+    Examples
+    --------
+
+    >>> import pandas as pd
+    >>> import texthero as hero
+    >>> s = pd.Series(["he couldd spellit correctlly"])
+    >>> hero.correct_mistakes(s)
+    0    he could spell it correctly
+    dtype: object
+    >>> s = pd.Series(["thequickbrownfoxjumpsoverthelazydog"])
+    >>> hero.correct_mistakes(s, True)
+    0    the quick brown fox jumps over the lazy dog
+    dtype: object
+    
+    """
+    sym_spell = SymSpell(max_dictionary_edit_distance=3, prefix_length=7)
+    dictionary_path = pkg_resources.resource_filename("symspellpy", "frequency_dictionary_en_82_765.txt")
+    bigram_path = pkg_resources.resource_filename("symspellpy", "frequency_bigramdictionary_en_243_342.txt")
+
+    sym_spell.load_dictionary(dictionary_path, term_index=0, count_index=1)
+    sym_spell.load_bigram_dictionary(bigram_path, term_index=0, count_index=2)
+
+    if fix_spacing is True:
```

Comment:
I think the is in python does just compare if two objects are identical. This won't fail here, since True is a basic type, but just `if fix_spacing:` would be more elegant